src/commands/initProjectForVSCode/InitVSCodeLanguageStep.ts

```diff
@@ -22,21 +22,24 @@ export class InitVSCodeLanguageStep extends AzureWizardPromptStep<IProjectWizard
 
     public async prompt(context: IProjectWizardContext): Promise<void> {
         // Display all languages, even if we don't have full support for them
-        const languagePicks: QuickPickItem[] = [
-            { label: ProjectLanguage.CSharp },
-            { label: ProjectLanguage.CSharpScript },
-            { label: ProjectLanguage.FSharp },
-            { label: ProjectLanguage.FSharpScript },
-            { label: ProjectLanguage.Java },
-            { label: ProjectLanguage.JavaScript },
-            { label: ProjectLanguage.PowerShell },
-            { label: ProjectLanguage.Python },
-            { label: ProjectLanguage.TypeScript },
-            { label: ProjectLanguage.Custom }
+        const languagePicks: IAzureQuickPickItem<{ language: ProjectLanguage, model?: number }>[] = [
```

Comment:
This `{ language: ProjectLanguage, model?: number }` "type" is used in a couple places; it might benefit from a proper type definition.

Reply:
How about, if I see it in one more place, I promise to do so. :-)